src/sql/src/plan.rs

```diff
@@ -794,7 +794,7 @@ pub struct SetTransactionPlan {
 
 #[derive(Clone, Debug)]
 pub struct SelectPlan {
-    pub select: Option<SelectStatement<Aug>>,
+    pub select: Option<Box<SelectStatement<Aug>>>,
```

Comment:
Mind tossing a comment here explaining the `Box` is intentional? (sorry I should have mentioned this on the previous PR as well)